tests/tests.js

```diff
@@ -26,7 +25,21 @@ else {
     if (actual != expected) throw new Error(JSON.stringify(actual) + ' == ' + JSON.stringify(expected));
   };
   assert.deepEqual = function(actual, expected) {
-    if (!_.isEqual(actual, expected)) throw new Error(JSON.stringify(actual) + ' == ' + JSON.stringify(expected));
+    var actual_keys = Object.keys(actual), expected_keys = Object.keys(expected);
+    var has_error = false;
+    if (actual_keys.length != expected_keys.length) 
+      has_error = true;
+    
+    actual_keys.forEach(function(key) {
+      if (actual[key] != expected[key]) has_error = true;
+    });
+
+    expected_keys.forEach(function(key) {
+      if (actual[key] != expected[key]) has_error = true;
+    });
+
+    if (has_error)
+      throw new Error(JSON.stringify(actual) + ' == ' + JSON.stringify(expected));

```

Comment:
@schuttsm: Looks like this is a shallow isEqual (doesn't recurse into nested properties)... but maybe that's all we need? The underscore implementation is a bit long, the [npm module](https://github.com/inspect-js/node-deep-equal) may be even longer.

It might help to stick a breakpoint in & see how much it's used & what types of inputs we're sending it (looks like most of the tests are not using it & are just comparing SQL strings).